src/output/file.rs

```diff
@@ -52,14 +54,17 @@ impl File {
     /// ```
     /// # use ocptv::output::*;
     ///
-    /// let file = File::new("name", "/path/to/file", false);
+    /// let uri = Uri::parse("file:///tmp/foo").unwrap();
+    /// let file = File::new("name", uri);
     /// ```
-    pub fn new(name: &str, uri: &str, is_snapshot: bool) -> Self {
+    pub fn new(name: &str, uri: tv::Uri) -> Self {
         File {
             name: name.to_owned(),
-            uri: uri.to_owned(),
-            is_snapshot,
-            ..Default::default()
+            uri,
+            is_snapshot: false,
```

Comment:
what happened with the `..Default::default()`? did i remove it in the branch fix?

Reply:
Nope, the problem is Uri that doesn't have a default